.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -55,7 +57,7 @@ jobs:
           CIBW_ARCHS_LINUX: ${{matrix.arch}}
 
       - uses: actions/upload-artifact@v2
-        if: github.ref == 'refs/heads/master'
+        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' }}
```

Comment:
Why do we need the `or` here?

Reply:
Good question. I think `||` is supported (link https://docs.github.com/en/actions/learn-github-actions/expressions#operators) though happy to modify if you think `or` is better.

Reply:
Ah, I misunderstood. Yea, we want this to run in both cases. We can issue a release from a non master branch (the tagged v0.xy branch), and ensure the wheels are built.  Master may have since move on, so we need to be able to support both ( I think).